be/src/exec/data_sink.cpp

```diff
@@ -152,18 +153,18 @@ Status DataSink::create_data_sink(RuntimeState* state, const TDataSink& thrift_s
         *sink = std::make_unique<IcebergTableSink>(state->obj_pool(), output_exprs);
         break;
     }
-
-    default:
-        std::stringstream error_msg;
-        auto i = _TDataSinkType_VALUES_TO_NAMES.find(thrift_sink.type);
-        const char* str = "Unknown data sink type ";
-
-        if (i != _TDataSinkType_VALUES_TO_NAMES.end()) {
-            str = i->second;
+    case TDataSinkType::TABLE_FUNCTION_TABLE_SINK: {
```

Comment:
It's better to add a clear commit message in the pull request.

Reply:
added some description for this PR